src/DatabaseWrapper.Oracle/OracleHelper.cs

```diff
@@ -0,0 +1,1387 @@
+                    }
+                    break;
+                case DataTypeEnum.Blob:
+                    ret += "blob  ";
+                    break;
+                
+                
+                default:
+                    throw new ArgumentException("Unknown DataType: " + col.Type.ToString());
+            }
+
+            if (col.Nullable) ret += "NULL ";
+            else ret += "NOT NULL ";
+
+            //if (col.PrimaryKey) ret += "AUTO_INCREMENT ";
+
+            return ret;
+        }
+
+        /// <summary>
+        /// Retrieve the primary key column from a list of columns.
+        /// </summary>
+        /// <param name="columns">List of Column.</param>
+        /// <returns>Column.</returns>
+        public override Column GetPrimaryKeyColumn(List<Column> columns)
+        {
+            Column c = columns.FirstOrDefault(d => d.PrimaryKey);
+            if (c == null || c == default(Column)) return null;
+            return c;
+        }
+
+        /// <summary>
+        /// Retrieve a query used for table creation.
+        /// </summary>
+        /// <param name="tableName">Table name.</param>
+        /// <param name="columns">List of columns.</param>
+        /// <returns>String.</returns>
+        public override string CreateTableQuery(string tableName, List<Column> columns)
+        {
+            
+            string query =
+                "CREATE TABLE " + SanitizeString(tableName) + " " +
+                "(";
+
+            int added = 0;
+            foreach (Column curr in columns)
+            {
+                if (added > 0) query += ", ";
+                query += ColumnToCreateQuery(curr);
+                added++;
+            }
+
+            Column primaryKey = GetPrimaryKeyColumn(columns);
+            if (primaryKey != null)
+            {
+                query +=
+                    "," +
+                    "PRIMARY KEY (" + SanitizeString(primaryKey.Name) + ")";
+            }
+
+            query +=
+                ") ";
+            if (primaryKey != null) {
+                //query += $"CREATE SEQUENCE {tableName}_seq START WITH 1 INCREMENT BY 1;" +
+                //           $"CREATE OR REPLACE TRIGGER {tableName}_seq_tr " +
+                //           "BEFORE INSERT ON person FOR EACH ROW " +
+                //           "WHEN (NEW.id IS NULL) " +
+                //           "BEGIN " +
+                //           "SELECT person_seq.NEXTVAL INTO :NEW.id FROM DUAL; " +
+                //           "END; ";
+
+
+            }
+            return query;
+        }
+        public string CreateSequnce(string tableName) 
+        {
+            string query = $"CREATE SEQUENCE {tableName}_seq START WITH 1 INCREMENT BY 1";
+            return query;
+        }
+        public string CreateTrigger(string tableName)
+        {
+            string query = $"CREATE OR REPLACE TRIGGER {tableName}_seq_tr " +
+                           "BEFORE INSERT ON person FOR EACH ROW " +
```

Comment:
Changing this to {tableName} instead of `person` - should the `id` also be an input parameter?  Assuming so